Source/ACE/Entity/WorldObject.cs

```diff
@@ -160,101 +160,101 @@ public virtual void SerializeCreateObject(BinaryWriter writer)
             if ((WeenieFlags & WeenieHeaderFlag.PuralName) != 0)
                 writer.WriteString16L(GameData.NamePlural);
 
-            if ((WeenieFlags & WeenieHeaderFlag.ItemCapacity) != 0)
-                writer.Write(GameData.ItemCapacity);
+            if (((WeenieFlags & WeenieHeaderFlag.ItemCapacity) != 0) && GameData.ItemCapacity != null)
```

Comment:
I must just be missing what you guys are driving at - can either of you give a concrete example?   If the flag for Item Capacity is set - you have to send something non null.   If you unset the flag - it does not matter what is in gamedata.ItemCapacity - it is not going to get sent to the client.   That is how it was working before I did anything to it.   The flag controls the write on our end and tells the client what to expect to read on the client side.   Sorry for being obtuse - but I don't understand what you guys are driving at.